### DISCUSSION TITLE
Dependency Dashboard gets cut off

### DISCUSSION THREAD

**Discussion by gnohj** [renovatebot/renovate#20300] [Request Help] [answered]
### I'm using the Mend Renovate hosted app.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe your problem.

**Pending Approval**
................... (omitted for brevity, but shows full list)

**Awaiting Schedule**
```
 ................
 fix(deps): update dependency *** to v5.3.0
 fix(deps): update dependency *** to v5.3.0
 <!-- unschedule-branch=ren
```

This is the last line from RenovateBot. It stops at **Awaiting Schedule Section** and gets cut off.
I dont see the Rate-Limited, Open, Ignored or Blocked, Detected Dependencies, or  a Checkbox for retrigger.
`renovate-config-revalidator` shows config validation as success.
Scheduled PRs are getting created correctly. Only issue is the dashboard cut off.

### Relevant debug logs.

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment]
probably because you've too many PR's waiting for approval.

**Comment by HonkingGoose** [comment, marked as answer]
Related issues:

- https://github.com/renovatebot/renovate/issues/18244
- https://github.com/renovatebot/renovate/issues/17174
